Description: Modified the build procedure.
 Added make.sh and Makefile and modified build.sh

--- /dev/null
+++ syncthing-0.8.19/Makefile
@@ -0,0 +1,5 @@
+default:
+	sh make.sh
+
+install:
+	echo "Installation happens further down."
--- syncthing-0.8.19.orig/build.sh
+++ syncthing-0.8.19/build.sh
@@ -24,7 +24,7 @@ build() {
 
 	go vet ./...
 
-	godep go build $* -ldflags "$ldflags" ./cmd/syncthing
+	godep go build -v -x $* ./cmd/syncthing #-gccgoflags "$ldflags"
 }
 
 assets() {
--- /dev/null
+++ syncthing-0.8.19/make.sh
@@ -0,0 +1,21 @@
+#!/bin/sh
+echo "Setting GOPATH, GOROOT and GOBIN..."
+export GOPATH=$(pwd)/gopath
+mkdir -p $GOPATH/root
+export GOROOT=$GOPATH/root
+mkdir -p $GOPATH/bin
+export GOBIN=$GOPATH/bin
+export PATH=$PATH:$GOBIN
+echo "Installing godep..."
+go install github.com/tools/godep
+echo "Installing go vet..."
+go install code.google.com/p/go.tools/cmd/vet
+echo "Creating a symbolic link..."
+mkdir -p $GOPATH/src/github.com/calmh
+rm -f $GOPATH/src/github.com/calmh/syncthing
+ln -s $(pwd) $GOPATH/src/github.com/calmh/syncthing
+
+echo "Building it..."
+rm syncthing
+./build.sh
+
